.github/workflows/checks.yml

```diff
@@ -52,9 +52,9 @@ on:
   merge_group:
     types: ["checks_requested"]
   push:
-    branches: ["nightly", "devnet-freeze", "main"]
+    branches: ["nightly", "devnet-freeze", "release-v*"]
```

Comment:
This is matching a substring?

If the release is called `release-v1.1.1` would this work?

Reply:
Oh nevermind, it was hard to see the `*` at the end.